packages/now-cli/src/util/dev/server.ts

```diff
@@ -1767,30 +1782,26 @@ function proxyPass(
   req: http.IncomingMessage,
   res: http.ServerResponse,
   dest: string,
+  proxy: httpProxy,
   output: Output,
   ignorePath: boolean = true
 ): void {
-  const proxy = httpProxy.createProxyServer({
-    changeOrigin: true,
-    ws: true,
-    xfwd: true,
-    ignorePath,
-    target: dest,
-  });
+  return proxy.web(
+    req,
+    res,
+    { target: dest, ignorePath },
+    (error: NodeJS.ErrnoException) => {
+      // If the client hangs up a socket, we do not
+      // want to do anything, as the client just expects
+      // the connection to be closed.
+      if (error.code === 'ECONNRESET') {
+        res.end();
+        return;
+      }
 
-  proxy.on('error', (error: NodeJS.ErrnoException) => {
-    // If the client hangs up a socket, we do not
-    // want to do anything, as the client just expects
-    // the connection to be closed.
-    if (error.code === 'ECONNRESET') {
-      res.end();
-      return;
+      output.error(`Failed to complete request to ${req.url}: ${error}`);
     }
```

Comment:
I think I was able to verify this refactor, by closing a tab and confirming that I _didn't_ see "Failed to complete request…" in the debug output?